examples/onnxrt/nlp/huggingface_model/code_detection/quantization/ptq_static/main.py

```diff
@@ -0,0 +1,347 @@
+
+    tokenizer = AutoTokenizer.from_pretrained(model_name_or_path)
+    encodings = tokenizer(
+        texts, return_tensors="pt", truncation=True, padding="max_length"
+    )
+
+    class CodeDataset(torch.utils.data.Dataset):
+        def __init__(self, encodings, labels):
+            self.encodings = encodings
+            self.labels = labels
+
+        def __getitem__(self, idx):
+            item = {key: torch.tensor(val[idx]) for key, val in self.encodings.items()}
+            item["labels"] = torch.tensor(self.labels[idx])
+            return item
+
+        def __len__(self):
+            return len(self.labels)
+
+    dataset = CodeDataset(encodings, labels)
+    return dataset
+
+
+# evaluation func for fine-tuning
+def evaluate(model, val_loader):
+    import torch
+
+    print("***　eval model .. ")
+    all_labels = []
+    all_preds = []
+    for idx, batch in enumerate(val_loader):
+        model.eval()
+        with torch.no_grad():
+            labels = batch.pop("labels")
+            inputs = batch
+            outputs = model(**inputs, labels=labels)
+            loss = outputs.loss
+            logits = outputs.logits
+            all_labels.append(labels.numpy())
+            all_preds.append(np.argmax(logits.detach().numpy(), axis=1))
+            np.concatenate(all_labels, axis=0)
+            np.concatenate(all_preds, axis=0)
+            cur_acc = np.mean(
+                np.concatenate(all_labels, axis=0) == np.concatenate(all_preds, axis=0)
+            )
+            print(f"{idx} batch evaluation accuracy: {cur_acc}")
+    cur_acc = np.mean(
+        np.concatenate(all_labels, axis=0) == np.concatenate(all_preds, axis=0)
+    )
+    print("Overall evaluation accuracy: ", cur_acc)
+    return cur_acc
+
+
+def fine_tune(args):
+    import os
+
+    import numpy as np
+    import torch
+    from torch.utils.data import DataLoader
+    from transformers import AdamW, AutoModelForSequenceClassification
+
+    train_dataset = load_dataset_from_local(
+        args.train_data_path, args.model_name_or_path
+    )
+    val_dataset = load_dataset_from_local(args.data_path, args.model_name_or_path)
+    model = AutoModelForSequenceClassification.from_pretrained(args.model_name_or_path)
+    val_loader = DataLoader(val_dataset, batch_size=64, shuffle=True)
+    train_loader = DataLoader(train_dataset, batch_size=64, shuffle=True)
+
+    optim = AdamW(model.parameters(), lr=5e-5)
+
+    results = {"eval_acc": 0}
+    global_step = -1
+    for epoch in range(3):
+        all_labels = []
+        all_preds = []
+        for idx, batch in enumerate(train_loader):
+            global_step += 1
+            optim.zero_grad()
+            labels = batch.pop("labels")
+            inputs = batch
+            model.train()
+            outputs = model(**inputs, labels=labels)
+            loss = outputs.loss
+            logits = outputs.logits
+            # print(np.argmax(logits.detach().numpy(), axis=1))
```

Comment:
remove the print

Reply:
Removed by [0c26397](https://github.com/intel/neural-compressor/pull/1121/commits/0c2639750f33f6b306544dba11175291c4ad691d).